gensrc/script/functions.py

```diff
@@ -1210,6 +1210,9 @@
     [150334, 'array_generate', True, False, 'ARRAY_LARGEINT', ['LARGEINT', 'LARGEINT', 'LARGEINT'],
      "ArrayFunctions::array_generate<TYPE_LARGEINT>"],
 
+    [150335, 'array_sortby', True, False, 'ANY_ARRAY', ['ANY_ARRAY', 'ANY_ARRAY', 'ANY_ARRAY', "..."]
+    ,'ArrayFunctions::array_sortby<TYPE_BOOLEAN>'],
```

Comment:
It's better to create some new functions: `ArrayFunctions::array_sortby_multi` ⇨ `ArraySortByMulti::process`. 

`array_sortby<TYPE_BOOLEAN>` may be misunderstanding, because `TYPE_BOOLEAN` is useless, and the logic of `array_sortby_multi` and `array_sortby` are totally different.